src/test/java/api/support/fixtures/OpeningPeriodsExamples.java

```diff
@@ -0,0 +1,83 @@
+package api.support.fixtures;
+
+import api.support.builders.OpeningPeriodsBuilder;
+import org.folio.circulation.domain.OpeningDay;
+import org.folio.circulation.domain.OpeningHour;
+import org.folio.circulation.domain.OpeningPeriod;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OpeningPeriodsExamples {
```

Comment:
It seems to me that it is more like data provider for your class. In circulation, `*Examples` commonly means a class that provides sample entities, `OpeningPeriods` as in this case.

Should it be renamed if it is a data provider or refactored as following, if it is intended to be a circ example class:
* Do not rely on ServicePoint id;
* Instead have methods that returns
  * twoOpeningDaysPeriod
  * allDayOpenningDaysPeriod
  * etc.
* `CASE_ERROR_400_SERVICE_ID`, `CASE_ERROR_404_SERVICE_ID`, `CASE_ERROR_500_SERVICE_ID` should be moved to test class, they sounds to me like test specific.

Reply:
moved all the examples to the test class